src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -384,14 +384,17 @@ public function get_result_messages( $queue ) {
 										 _x( ' at ', 'separator between date and time', 'the-events-calendar' ) .
 										 date( get_option( 'time_format' ), $scheduled_time );
 
-				$messages['success'][] = '<br/>' .
-										 sprintf( // add in timing
-											 __( 'The next import is scheduled for %1$s.', 'the-events-calendar' ),
-											 esc_html( $scheduled_time_string )
-										 ) .
-										 ' <a href="' . admin_url( 'edit.php?page=aggregator&post_type=tribe_events&tab=scheduled' ) . '">' .
-										 __( 'View your scheduled imports.', 'the-events-calendar' ) .
-										 '</a>';
+                if ( 'on_demand' !== $queue->record->frequency->id ) {
+
+                    $messages['success'][] = '<br/>' .
+                                             sprintf( // add in timing
+                                                 __( 'The next import is scheduled for %1$s.', 'the-events-calendar' ),
+                                                 esc_html( $scheduled_time_string )
+                                             ) .
+                                             ' <a href="' . admin_url( 'edit.php?page=aggregator&post_type=tribe_events&tab=scheduled' ) . '">' .
+                                             __( 'View your scheduled imports.', 'the-events-calendar' ) .
+                                             '</a>';
+                }
```

Comment:
if it doesn't interfere with output, i would recommend putting this all in one `sprintf()` statement:

```
sprintf(
	__( 'The next import is scheduled for %1$s. <a href="%2$s">View your scheduled imports</a>' ),
	esc_html( $scheduled_time_string ),
	admin_url( 'edit.php?page=aggregator&post_type=tribe_events&tab=scheduled' )
);
```

Reply:
Since the scope of this ticket is to remove the message, I would rather not create more issues by changing the message and make this more complex to translate.

Reply:
okay 👍

Reply:
approved and ready for merge!